### DISCUSSION TITLE
Unsubscribe from ci-admin@eclipse.org

### DISCUSSION THREAD

**Discussion by iils-jschmidt** [eclipse-passage/passage#1315] [Q&A]
Hi there, 

since I commited to passage I receive mails from ci-admin@eclipse.org informing me about  the build status. I have not found any way to unsubscribe from this mailinglist. Where and how do I do this?

Best regards
Jens

**Comment by ruspl-afed** [comment]
Some of us use these emails to ensure that the Earth is still rolling. However, this should not be accompanied by collateral damage. Please let me know which Jenkins jobs are bothering you, and I will disable email notifications for them.
